CHANGELOG.md

```diff
@@ -5,6 +5,17 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/)
 and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.html).
 
+## feature/rds-phase-2
+
+## Breaking Changes
+
+This version of the dashboard requires Cumulus API release from feature/rds-phase-2
```

Comment:
The only thing is I'm wondering if we should wait until the branch is released then we'll have a specific version to mention